Improve docs (#21)
Quantco/metalearners: docs/glossary.rst

    Similar to the R-Learner, the Double Machine Learning blueprint
    relies on estimating two nuisance models in its first stage: a
    propensity model as well as an outcome model. Unlike the

>     R-Learner, the last-stage or treatment effect model might need to be a
>     specific type of estimator.

Reviewer: ```suggestion
    R-Learner, the last-stage or treatment effect model can be a
    any regression estimator.
```

Reviewer: :)

Author: I think the point we want to make here is that in DoubleML the last stage model may need to be of some predefined type, contrary to the RLearner where the last stage model can be any estimator.

Reviewer 2: I see; that's perfect.
